src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqUserRepository.kt

```diff
@@ -1,7 +1,6 @@
 package org.vaccineimpact.api.app.repositories.jooq
```

Comment:
There's still a chunk of duplicated logic between the two `getUser` methods. Surely the only thing that needs to differ between them is the `WHERE` clause?

Reply:
Not quite (one allows returning null, the other doesn't), but have changed `getUserByEmail...` to reuse logic from the other